pkg/cloudprovider/providers/gce/gce.go

```diff
@@ -87,6 +87,34 @@ type GCECloud struct {
 	nodeInstancePrefix       string   // If non-"", an advisory prefix for all nodes in the cluster
 	useMetadataServer        bool
 	operationPollRateLimiter flowcontrol.RateLimiter
+	manager                  ServiceManager
+}
+
+type ServiceManager interface {
+	// Creates a new persistent disk on GCE with the given disk spec.
+	CreateDisk(
```

Comment:
To conform with the rest of the files, can you put all the arguments on the same line?